### DISCUSSION TITLE
Pimcore\\Model\\Element\\Service::isValidKey(): Argument #1 ($key) must be of type string,

### DISCUSSION THREAD

**Discussion by chili-666** [pimcore/pimcore#16250] [Q&A / Support] [answered]
Hey there,

I am trying to generate an asset on my pimcore instance via the Datahub GraphgQL-Api. Unfortunately I get the following error:

`	"debugMessage": "Pimcore\\Model\\Element\\Service::isValidKey(): Argument #1 ($key) must be of type string, null given, called in \/home\/users\/username\/www\/pim.domain.tld\/vendor\/pimcore\/pimcore\/models\/Asset.php on line 627",`

I am using the following JSON for my request: 
`mutation {
  createAsset(
    parentId: 11
    filename: "foo1.jpg"
    type: "image"
    input: {
      data: "DATA"
    }
  ) {
    success
    message
    assetData {id}
  }
}`

I have a second pimcore up and running (setup only a day or two earlier) for a different purpose and there the request works like a charm. 

Any ideas where that error comes or where the error in my config might be?

Thanks in advance!

**Comment by kingjia90** [comment, marked as answer]
Should be fixed by https://github.com/pimcore/data-hub/pull/798
Looks like the error reported on https://github.com/pimcore/data-hub/issues/791

Please check if data-hub version is `1.6.4`
